package.json

```diff
@@ -21,61 +35,42 @@
     }
   ],
   "files": [
-    "dist/",
-    "src/",
-    "postinstall.js"
+    "src/"
```

Comment:
> At that point wouldn't they jus script incle to dist/min so package.json would have no effect

Come again? If they `npm install cross-domain-utils` they're not going to get the minified file in `dist/` unless we specify `dist/` here.

Also we still need to include `dist/module`, right? So it makes sense to include `dist/` here and just allow everything through.